balanced-parentheses/balanced-parentheses-helper.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Check if expression has balanced paranthesis.
+ * @param {string} expression The first number.
+ * @return {boolean} The sum of the two numbers.
+ */
+export const isBalanced = (expression) => {
+  console.log("EXPRESSION TO CHECK: ", expression);
+  if (!expression || expression.length == 0) return false;
+
+  const stack = [];
+  const ParanthesisMap = {
+    "(": ")",
+    "{": "}",
+    "[": "]",
+  };
+  const closeParantheses = [")", "}", "]"];
+
+  [...expression].every((item) => {
+    if (item in ParanthesisMap) stack.push(item);
+    else if (
```

Comment:
why are we using else if?

Reply:
because we only want to test one condition, we can use if if need to test both conditions